client/my-sites/email/email-management/email-home.tsx

```diff
@@ -68,6 +69,7 @@ interface EmailManagementHomeProps {
 	emailListInactiveHeader?: ReactElement;
 	sectionHeaderLabel?: TranslateResult;
 	selectedDomainName: string;
+	selectedIntervalLength: IntervalLength;
```

Comment:
Shouldn't this accept an `undefined` value as that is a valid return value from `castIntervalLength()`?
```suggestion
	selectedIntervalLength?: IntervalLength;
```